README.md

```diff
@@ -45,12 +52,33 @@ Using the Cluster API Provider Hetzner (CAPH) unites the benefits of declarative
 * Support for Hetzner Cloud placement groups, network and load balancer
 * Complete day 2 operations - updating Kubernetes and nodes, scaling up and down, self-healing
 * Custom CSR approver for approving [kubelet-serving certificate signing requests](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#kubelet-serving-certs)
-* Hetzner dedicated servers / bare metal
+* Hetzner dedicated servers / bare metal (and GPUs)
+
+## :eyes: Clarifying Scope
 
+Managing a production-grade Kubernetes system requires a **seasoned team of experts**.
 
+The Cluster API Provider Hetzner (CAPH) plays a specific, limited role within this intricate ecosystem. It deals with the low-level lifecycle management of machines and infrastructure. While it can support the creation of production-ready Kubernetes clusters, it does not define the specific contents within the cluster. This important task is something that needs to be handled independently.
```

Comment:
the specific content?

Reply:
@batistein about ".. it does not define the specific contents within the cluster." This wording was from a different PR. Can you elaborate?

Reply:
I just don't understand what is meant by it... Maybe we can change it. But probably doesn't block this